metaserver/src/test/scala/tests/search/SymbolIndexTest.scala

```diff
@@ -177,6 +177,8 @@ object SymbolIndexTest extends MegaSuite {
         assertSymbolFound(5, 5)("_root_.scala.collection.immutable.List.")
       "<<CharRef>>.create(...)" - // JavaMtags
         assertSymbolFound(8, 19)("_root_.scala.runtime.CharRef.")
+      "<<Either>>[...]" -
+        assertSymbolFound(10, 15)("_root_.scala.util.Either#")
```

Comment:
The expected symbol should be
```
_root_.scala.`package`.Either#
```
which is a type alias. However, I think we index that symbol as `_root_.scala.Either` in scalamtags. We need to add `package` to the owner when entering package objects in https://github.com/scalameta/language-server/blob/1299a44633f5f36f757ad26a08a52f53bc712b6c/metaserver/src/main/scala/scala/meta/languageserver/mtags/ScalaMtags.scala#L26

Reply:
A better way to test this is to update this line here 
https://github.com/scalameta/language-server/blob/1299a44633f5f36f757ad26a08a52f53bc712b6c/metaserver/src/test/scala/tests/mtags/ScalaMtagsTest.scala#L42
to expect `_root_.a.b.c.K.package.l.`